src/conn/conn_log.c

```diff
@@ -125,15 +125,23 @@ __logmgr_version(WT_SESSION_IMPL *session, bool reconfig)
 	}
 
 	/*
-	 * Set up the minimum log version required if needed.
+	 * Set up the maximum and minimum log version required if needed.
 	 */
-	if (conn->compat_req_major != WT_CONN_COMPAT_NONE) {
-		if (conn->compat_req_major < WT_LOG_V2_MAJOR)
-			conn->log_req_version = 1;
-		else if (conn->compat_req_minor == WT_LOG_V2_MINOR)
-			conn->log_req_version = 2;
+	if (conn->req_max_major != WT_CONN_COMPAT_NONE) {
+		if (conn->req_max_major < WT_LOG_V2_MAJOR)
+			conn->log_req_max = 1;
+		else if (conn->req_max_minor == WT_LOG_V2_MINOR)
```

Comment:
Now that we use both major and minor versions the comment in log.h [https://github.com/wiredtiger/wiredtiger/blob/518da6fac9bee33247d6cf879e68540cb37e9adf/src/include/log.h#L371](url) is not true any more.